Andri/wave background in flow

LXIF opened the pull request:
# Motivation

We want to delight our users with flair.

# Changes

Add flair canvas component, add it to our login screen with animation on login/signup success

<video src="https://github.com/user-attachments/assets/96193667-cd36-4cdf-a906-69ffec8c456f" />


# Tests

- [x] Tried signup flow
- [x] Tried signin flow

lmuntaner commented on src/frontend/src/routes/(new-styling)/authorize/+page.svelte:
```diff
@@ -19,9 +21,29 @@
+    }
+    setTimeout(async () => {
+      await goto("/authorize/account");
+    }, 700);
```
Ditto to convert to constant: `DELAY_ANIMATED_REDIRECT_MILLIS`

lmuntaner reviewed (commented):
Looking super good!

As a more architectural question, could we split where the component is rendered and where the animation is triggered?

It would be good that it works like the toaster component. It's always there and any component can trigger it.

Not necessarily in this PR, but as a comment for later.

LXIF commented:
Adjusted parameters and animations as per UI designer guidance. The signin/authorize flows now look like this:

<video src="https://github.com/user-attachments/assets/2fb1b8fd-5918-4356-8eb7-ff08d6f5a4bd" />

<video src="https://github.com/user-attachments/assets/867091c9-b1ab-42da-8c97-15d62401fa34" />

Also, I have added a feature flag called `FLAIR` which enables/disables the FlairCanvas inside the WaveCanvas.

LXIF commented on src/frontend/src/routes/(new-styling)/authorize/+layout.svelte:
```diff
@@ -62,17 +82,63 @@
+        impulseEasing: "cubicIn",
+      });
+    }
+    setTimeout(animateAuthenticating, 1200);
```
Removed, it was for loading state but was decided to not be there.
